08-projects.Rmd

```diff
@@ -10206,19 +10202,19 @@ The next couple of questions should be complete in the same language. You can us
 ```{block, type="bbox"}
 **Item(s) to submit:**
 
-- The R or Python code used to generate the graphic.
+- The R code used to generate the graphic.
 - The graphic in a png or jpg/jpeg format.
 ```
 
-##### 4. Use any combination of SQL, R, and Python you'd like in order to find which of the following 3 genres has the highest average rating for movies (see `type` column from `titles` table): Romance, Comedy, Animation. In the `titles` table, you can find the genres in the `genres` column. There may be some overlap (i.e. a movie may have more than one genre), this is ok. 
+##### 4. Now we will turn our focus to other information in the database. Use a combination of SQL and R to find which of the following 3 genres has the highest average rating for movies (see `type` column from `titles` table): Romance, Comedy, Animation. In the `titles` table, you can find the genres in the `genres` column. There may be some overlap (i.e. a movie may have more than one genre), this is ok. 
```

Comment:
I added the first sentence to clarify that we are moving the focus from _Friends_ to the database.

Reply:
Good idea